modules/convert/issue_comment.go

```
@@ -22,3 +22,132 @@ func ToComment(c *models.Comment) *api.Comment {
 		Updated:  c.UpdatedUnix.AsTime(),
 	}
 }
+
+// ToTimelineComment converts a models.Comment to the api.TimelineComment format
+func ToTimelineComment(c *models.Comment, doer *models.User) *api.TimelineComment {
+	err := c.LoadMilestone()
+	if err != nil {
+		return nil
```

Many errors (and below) are silenced, it will make debug more difficult if there is something wrong.

Can we return the errors or log them?